recipes/tensorflow-lite/all/conandata.yml

```diff
@@ -1,34 +1,14 @@
 sources:
-  "2.6.0":
-    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.6.0.tar.gz"
-    sha256: "41b32eeaddcbc02b0583660bcf508469550e4cd0f86b22d2abe72dfebeacde0f"
-  "2.6.2":
-    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.6.2.tar.gz"
-    sha256: "e68c1d346fc3d529653530ca346b2c62f5b31bd4fcca7ffc9c65bb39ab2f6ed3"
-  "2.9.1":
```

Comment:
Why drop 2.9? it's pretty recent

Reply:
It simplifies things a bit - we can remove the MSVC patch, for example. Also, it seemed like there's no reason to keep old versions around, since they're kept in conan center anyway?

Reply:
It's better to push updates whenever possible.

It's pretty rare we drop them #13049 is really pushing it and I considering putting them back

Reply:
Oh yeah, the requirements differ too, so it would complicate things a bit to keep both. Certainly doable, though, so if you'd prefer that, I can bring it back.